public/data/IsratIJK.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "Israt Jahan Khan",
+    "bio": "Cat lover",
+    "avatar": "https://github.com/IsratIJK.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/IsratIJK",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/IsratIJK",
+        "icon": "twitter"
+      }
+    ]
+  }
```

```suggestion
}
```
please take a look that the aligmnet of the lines matches

---

I have done what you had asked for. Can you please check that out?